sql/updates/latest-dev.sql

```diff
@@ -1,2 +1,4 @@
 DROP FUNCTION IF EXISTS _timescaledb_internal.time_col_name_for_chunk(name,name);
 DROP FUNCTION IF EXISTS _timescaledb_internal.time_col_type_for_chunk(name,name);
+
+ALTER TABLE _timescaledb_catalog.continuous_aggs_invalidation_threshold RENAME COLUMN watermark TO threshold;
```

Comment:
@erimatnor you are right. I didn't consider that TimescaleDB relies on GETSTRUCT a lot (instead of heap_deform_tuple) and this will probably not work.